### ISSUE TITLE
Color maps containing different color schemes for palette types and grafmdients.

### ISSUE THREAD

**Issue by prjctimg** [prjctimg/huetiful#6]
These should be `Map` data structures that store precompiled key value pairs for different color schemes or themes. The `Map`s can be used with WCAG utilities from Culori to create themes that are compliant to Material Design Guidelines.
